### PR TITLE
include io metrics for cgroup2

### PR DESCRIPTION
**Pull request by sigilioso**

### CODE REVIEW

File: src/raw/metrics.go [hunk 1]
```diff
@@ -6,6 +6,11 @@ import (
 	"github.com/docker/docker/api/types"
 )
 
+const (
```
Comment (marcsanmi) [inline src/raw/metrics.go:9]: Should we define these constants in the `cgroupv2.go` where they are used?

**Review by marcsanmi — APPROVED** [review]
LGMT, just left a nit.

File: src/raw/metrics.go (hunk 1 above)
Comment (marcsanmi) [reply src/raw/metrics.go:9]: Liked this reasoning 👍

**Review by alvarocabanas — APPROVED** [review]
LGTM

**Pull request merged**